ui/app/[locale]/(authorized)/SessionRefresher.tsx

```diff
@@ -25,7 +25,7 @@ export function SessionRefresher() {
   // immediately trigger a check to deal with expired sessions when opening the app after a long time
   useLayoutEffect(() => {
     void doRefresh();
-  // eslint-disable-next-line react-hooks/exhaustive-deps
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
just a nit

Reply:
Yep, added intentionally to control re-renders.